Kickstarter-iOS/Views/Controllers/CommentDialogViewController.swift

```diff
@@ -111,7 +111,8 @@ internal final class CommentDialogViewController: UIViewController {
 
   fileprivate func animateTextViewConstraint(_ change: Keyboard.Change) {
     UIView.animate(withDuration: change.duration, delay: 0.0, options: change.options, animations: {
-      self.bottomConstraint.constant = self.view.frame.height - change.frame.minY
+      let keyboardOffset = self.view.frame.height - change.frame.height
+      self.bottomConstraint.constant = keyboardOffset + Styles.grid(2)
```

Comment:
The additional `Styles.grid(2)` is just to give it a tiny bit of padding beyond the frame of the keyboard.